src/lib/voiceSearchHelper/__tests__/index-test.ts

```diff
@@ -79,9 +85,9 @@ describe('VoiceSearchHelper', () => {
     voiceSearchHelper.toggleListening();
     expect(onStateChange).toHaveBeenCalledTimes(1);
     expect(voiceSearchHelper.getState().status).toEqual('askingPermission');
-    recognition.onstart();
+    listeners.start();
```

Comment:
We could add a comment to inform that we simulate the `start` event of the `SpeechRecognition`.

Reply:
https://github.com/algolia/instantsearch.js/pull/3779/commits/c611f9dddb53d997cc210f077e28712d399fa88d